### PR TITLE
CREATE TABLE LIKE copies hidden columns

### PR DESCRIPTION
**Pull request by ericharmeling**
Fixes https://github.com/cockroachdb/docs/issues/11380.

### CODE REVIEW

File: v21.2/create-table.md [hunk 1]
```diff
@@ -127,9 +127,9 @@ The following options are supported:
 To exclude specifiers, use the `EXCLUDING` keyword. Excluding specifiers can be useful if you want to use `INCLUDING ALL`, and exclude just one or two specifiers. The last `INCLUDING`/`EXCLUDING` keyword for a given specifier takes priority.
 
 {{site.data.alerts.callout_info}}
-Column families, partitioning, interleavings, and foreign key constraints
-cannot be preserved from the old table and will have to be recreated
-manually in the new table if the user wishes.
+`CREATE TABLE LIKE` statements cannot copy [column families](column-families.html), [partitions](partitioning.html), and [foreign key constraints](foreign-key.html) from old tables. If you want these column qualifications in a new table, you must recreate them manually.
```
Comment (stbof) [inline v21.2/create-table.md:130]:
```suggestion
`CREATE TABLE LIKE` statements cannot copy [column families](column-families.html), [partitions](partitioning.html), and [foreign key constraints](foreign-key.html) from other tables. If you want these column qualifications in a new table, you must recreate them manually.
```

File: v21.2/create-table.md [hunk 2]
```diff
@@ -127,9 +127,9 @@ The following options are supported:
...
+
+`CREATE TABLE LIKE` copies all hidden columns (e.g., the hidden [`crdb_region`](set-locality.html#crdb_region) column in multi-region databases) from old tables to new tables.
```
Comment (stbof) [inline v21.2/create-table.md:132]:
```suggestion
`CREATE TABLE LIKE` copies all hidden columns (e.g., the hidden [`crdb_region`](set-locality.html#crdb_region) column in multi-region databases) from the existing table to the new table.
```

**Review by stbof — APPROVED** [review]
I find the word `old` to be kind of strange. My suggested replacement is `existing`.

File: v21.2/create-table.md (hunk 1 above)
Comment (ericharmeling) [reply v21.2/create-table.md]: Done

File: v21.2/create-table.md (hunk 2 above)
Comment (ericharmeling) [reply v21.2/create-table.md]: Done

**Review by ericharmeling — COMMENTED** [review]
TFTR!

**Comment by ericharmeling** [comment]
> I find the word `old` to be kind of strange. My suggested replacement is `existing`.

I agree. I updated it here. Unfortunately, I think we use this kind of terminology in a lot of docs where we could use "existing" instead. I'll try to update it where I see it.

In general, using "old" could be perceived as non-inclusive (in addition to not being the most precise term), so I also added it to the ["Write about features and users in inclusive ways" section of the style guide](https://github.com/cockroachdb/docs/wiki/Style-Guide#write-about-features-and-users-in-inclusive-ways).

**Pull request merged**